sovtokenfees/sovtokenfees/req_handlers/write_handlers/xfer_fee_handler.py

```diff
@@ -0,0 +1,13 @@
+from sovtoken.constants import XFER_PUBLIC
+
+from indy_common.authorize.auth_actions import AuthActionAdd
+from sovtoken.request_handlers.write_request_handler.xfer_handler import XferHandler
+
+from plenum.common.request import Request
+
+
+class XferFeeHandler(XferHandler):
+    def dynamic_validation(self, request: Request):
```

Comment:
Please add a comment why we overriding this method

Reply:
ok